qemu/qemu.go

```diff
@@ -771,6 +771,9 @@ type Knobs struct {
 	// MemPrealloc will allocate all the RAM upfront
 	MemPrealloc bool
 
+	// HugePages will pre-allocate all the RAM from huge pages
+	HugePages bool
```

Comment:
Maybe we should add a note here, and to MemPrealloc, to say that if you set HugePages you should also set Memory.Size otherwise setting HugePages will have no effect.

Reply:
@markdryan will do. Also do we even work if the caller does not set Memory.Size? Do we choose a default anywhere. Maybe I missed that.

Reply:
Nevermind.. we allow QEMU to pick the default. Added a comment, that calls that out.